gcc/testsuite/lib/target-supports.exp

```diff
@@ -11843,3 +11843,17 @@ main:
 	.byte 0
   } ""]
 }
+
+# Return 1 if CORE-V architecture is available.
+
+proc check_effective_target_corev_mac { } {
+    if { !([istarget riscv*-*-*]) } {
+         return 0
+     }
+    return [check_no_compiler_messages corev_mac object {
+        void foo (void)
+        {
+          asm ("cv.mac t0, t1, t2");
+        }
+    } "-march=rv32ixcorevmac1p0" ]
```

Comment:
This is a separate issue - the architecture names all need updating to match the [agreed specification](https://github.com/openhwgroup/core-v-sw/blob/master/specifications/corev-isa-extension-naming.md).  In this case `xcvmac`, not `xcorevmac`.  Personally I don't think you need to specify the version number.

Reply:
As discussed offline, this needs to be coordinated with binutils-gdb and also cover all CORE-V extensions.  This can stand in this PR, and will be fixed in two overarching PRs for binutils-gdb and gcc.